client/src/data/locales/en-US.json

```
@@ -10,6 +10,7 @@
   "hero.photoCredit": "Photo by {photographerLink} on {platformLink}",
   "aboutUs.title": "Who We Are:",
   "aboutUs.weAre": "We are a collection of restaurant owners, bartenders, chefs, and patrons; <b>Just Like You</b>. We understand how hard it is navigating the red tape of liquor licenses in Boston especially for your first time.{br}From tracking availability, understanding the application process, to the potential hidden costs of obtaining one. There is a lot to keep track of. We’ve made it our mission to demystify the process, and help you get your business off the ground as painlessly as possible.",
+  "home.quote": "...working to create a more equitable restaurant ecosystem for all...",
```

Requested Changes: "Extra work" translation header got clobbered here. We want to keep it.
```suggestion
  "home.quote": "...working to create a more equitable restaurant ecosystem for all...",
  "home.extra-work.h2": "Check out more of our work!",
```